src/scripts/crowdin-import.ts

```diff
@@ -0,0 +1,348 @@
+// Library requires
+const { copyFileSync, existsSync, mkdirSync, readdirSync } = require("fs")
+const { resolve, join } = require("path")
+const argv = require("minimist")(process.argv.slice(2))
+
+/**
+ * Console flags
+ * -v,--verbose    Prints verbose console logs
+ * -f,--full       Prints full name of buckets in summary
+ *
+ *
+ * Follow these steps to import translations from Crowdin export:
+ *
+ * 1. Copy languages folder from Crowdin export to ./.crowdin
+ *    ie. ./.crowdin/{lang-codes}
+ *    Tip: Run `yarn crowdin-clean` to initialize the `.crowdin` folder. Can
+ *    also be used to erase contents when finished.
+ *
+ * 2. Select buckets to import by adding the number of the corresponding
+ *    content bucket to the chosen language array below
+ *    ie. `es: [1, 10],` would import the "Homepage" and "Learn" buckets for Spanish
+ *
+ * 3. Save file without committing \*
+ *
+ * 4. Execute script by running `yarn crowdin-import`
+ *
+ * 5. If successful, copy `GATSBY_BUILD_LANGS={langs}` output and paste in
+ *    your `.env`, then build site to test results.
+ *
+ * Remember: Revert working changes to this file before committing Crowdin import
+ */
+
+type UserSelectionObject = { [key: string]: number[] }
+const USER_SELECTION: UserSelectionObject = {
```

Comment:
More of a nice-to-have but perhaps we could generate this object from `src/utils/languages.ts` to ensure this stays in sync with the languages we support?

Reply:
@samajammin Yes... originally implemented this verbose hard-coded form intentionally to make it easy for users to just add numbers to the array without having to setup the object themself in any way. 

But, just left a comment above responding to Pablo where I have some ideas for v2, where we generate a git-ignored file using a third script (something like `crowdin-setup`) which could generate an object based on the TS langs available, making the full object still available for the user, just not in the `crowdin-imports.ts` file anymore.

This would solve this issue, and also the issue with asking users to alter a git-tracked file but to essentially try not to commit these changes.